.circleci/config.yml

```diff
@@ -382,6 +382,7 @@ workflows:
           build: npx cross-env QUICK_BUILD=true yarn run build
           # start server --> verify running --> percy + chrome + cypress
           command: yarn run test:e2e:dist
+          spec: 'cypress/integration/visual-regression/**/*'
```

Comment:
Looks like something else is missing. I ran the `PR_OPTIONAL_VISUAL_TESTS` and it getting the tests under `/integration/common/` and not from `/integration/visual-regression/`

Reply:
Fixed!